### PR TITLE
Fix description in IfStatement

### PR DESCRIPTION
**Pull request by brandboat**
Fix incorrect description in IfStatement, and replace it with the right method name in RecordCursor.

### CODE REVIEW

File: core/trino-main/src/main/java/io/trino/sql/gen/CursorProcessorCompiler.java [hunk 1]
```diff
@@ -305,9 +306,20 @@ public BytecodeNode visitInputReference(InputReferenceExpression node, Scope sco
...
+                String recordCursorMethodName;
+                if (javaType == boolean.class || javaType == long.class || javaType == double.class) {
+                    recordCursorMethodName = "get" + Primitives.wrap(javaType).getSimpleName();
+                }
+                else if (javaType == Slice.class) {
+                    recordCursorMethodName = "getSlice";
+                }
+                else {
+                    recordCursorMethodName = "getObject";
+                }
```
Comment (findepi) [inline core/trino-main/src/main/java/io/trino/sql/gen/CursorProcessorCompiler.java:318]: that's overkill if it is just for description purposes.

Comment (brandboat) [reply core/trino-main/src/main/java/io/trino/sql/gen/CursorProcessorCompiler.java:318]: Thanks for your comment !

> that's overkill 

Agree with this, but I don’t have a better idea to solve this problem... Any suggestions ?

> if it is just for description purposes.

Yes this is merely for fixing description.

Comment (findepi) [reply core/trino-main/src/main/java/io/trino/sql/gen/CursorProcessorCompiler.java:318]: what about
```
format("cursor.get<%s>(%d)", type, field)
```
?

Comment (brandboat) [reply core/trino-main/src/main/java/io/trino/sql/gen/CursorProcessorCompiler.java:318]: You mean no need to fix this ? If you think doing didn't bring more value, I can close this PR.

**Comment by mosabua** [comment]
:wave: @brandboat  - this PR has become inactive. If you're still interested in working on it, please let us know, and we can try to get reviewers to help with that.

We're working on closing out old and inactive PRs, so if you're too busy or this has too many merge conflicts to be worth picking back up, we'll be making another pass to close it out in a few weeks.

**Comment by brandboat** [comment]
Close this pr as it doesn't seem to bring much value

**Pull request closed without merging**